jenkins/ml-models.JenkinsFile

```diff
@@ -58,15 +82,50 @@ pipeline {
         stage('Download the artifacts') {
             steps {
                 script {
-                    downloadFromS3(
-                        assumedRoleName: 'get_models',
-                        roleAccountNumberCred: 'ml-models-aws-account-number',
-                        downloadPath: "${env.ARTIFACT_PATH}",
-                        bucketName: "${env.BUCKET_NAME}",
-                        localPath: "${env.WORKSPACE}/artifacts",
-                        force: true,
-                        region: 'us-west-2'
-                    )
+                    if (env.BASE_DOWNLOAD_PATH == "ml-models/model_listing") {
+                        downloadFromS3(
+                            assumedRoleName: 'get_models',
+                            roleAccountNumberCred: 'ml-models-aws-account-number',
+                            downloadPath: "${env.ARTIFACT_PATH}/",
+                            bucketName: "${env.BUCKET_NAME}",
+                            localPath: "${env.WORKSPACE}/artifacts",
+                            force: true,
+                            region: 'us-west-2'
+                        )
+                    } else {
+                        if (env.FORMAT == "TORCH_SCRIPT") {
+                            downloadFromS3(
+                                assumedRoleName: 'get_models',
+                                roleAccountNumberCred: 'ml-models-aws-account-number',
+                                downloadPath: "${env.ARTIFACT_PATH}/torch_script/",
+                                bucketName: "${env.BUCKET_NAME}",
+                                localPath: "${env.WORKSPACE}/artifacts",
+                                force: true,
+                                region: 'us-west-2'
+                            )
+                        } else if (env.FORMAT == "ONNX") {
+                            downloadFromS3(
+                                assumedRoleName: 'get_models',
+                                roleAccountNumberCred: 'ml-models-aws-account-number',
+                                downloadPath: "${env.ARTIFACT_PATH}/onnx/",
+                                bucketName: "${env.BUCKET_NAME}",
+                                localPath: "${env.WORKSPACE}/artifacts",
+                                force: true,
+                                region: 'us-west-2'
+                            )
+                        } else { 
```

Comment:
Should this be `else if (env.FORMAT == "BOTH")`

Reply:
We can do that as well. Will make that change.